clash_lib/src/proxy/utils/test_utils/docker_runner.rs

```diff
@@ -0,0 +1,158 @@
+//! This example will run a non-interactive command inside the container using `docker exec`
+
+use std::collections::HashMap;
+
+use bollard::container::{Config, RemoveContainerOptions};
+use bollard::secret::{HostConfig, Mount, PortBinding};
+use bollard::Docker;
+
+use bollard::exec::{CreateExecOptions, StartExecResults};
+use bollard::image::CreateImageOptions;
+
+use anyhow::Result;
+use futures::{Future, StreamExt, TryStreamExt};
+
+pub struct DockerTestRunner {
+    _instance: Docker,
```

Comment:
why `_`?

Reply:
Oh, there is a story. 
You see, but when I first create the DockerTestRunner, i add `DockerTestRunner::exec` to execute docker commands, it needs the docker instance, so the instance shall be kept in `DockerTestRunner`.
But then, i found that, actually, these docker images all provide an entrypoint, so it's ok to just pass the startup parameters and don't do the `docker exec` job. So i turn the `instance` into `_instance`. 

So shall i just delete the `_instance` & `DockerTestRunner::exec` to simplify the code?

Reply:
Yeah if the entrypoint/cmd does the thing already, we can just use them

Reply:
fixed